streamwebs_frontend/streamwebs/templates/streamwebs/resources.html

```diff
@@ -0,0 +1,47 @@
+{% extends "streamwebs/base.html" %}
+
+{% block content %}
+<div class="container">
+  <h3>
+    <strong>
+      StreamWebs offers a number of resources to get you started in the field.
+    </strong>
+  </h3>
+  <p><a href="data-sheets">DATA SHEETS</a>:
+  </br>
```

Comment:
`br` isn't a closing tag. It should look like either `<br>` or `<br />`. That's a choice of style, but `</br>` is incorrect.